src/admin/includes/formDefinitions/callbacks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+function processInsert(){
+  $localvars = localvars::getInstance();
+  $formInfo  = $_POST['MYSQL'];
```

Comment:
why are you assigning $_POST['MYSQL'] to a new variable? You can use $_POST['MYSQL'] directly. All this does is use extra memory in the application. The data is already in memory. it doesn't need to be in memory twice.